src/components/EventPage.js

```diff
@@ -3,38 +3,76 @@ import React, { Component } from "react";
 import { StyleSheet, View } from "react-native";
 
 // Vendor
-import { Button, Text } from "native-base";
+import {
+  Container, Header, Content, H1, H2, H3, Button, Text,
+} from 'native-base';
 
 // Redadalertas
 import { colors } from "styles";
 
 const styles = StyleSheet.create({
-  container: {
+  view: {
     flex: 1,
+    flexDirection: "row",
     justifyContent: "center",
-    alignItems: "center",
+    alignItems: "stretch",
     backgroundColor: "#F5FCFF",
     color: "#000000"
+  },
+  content: {
+    padding: 20
   }
 });
 
+const types = [
+  { label: "Raid", value: "sweep" },
+  { label: "Individual", value: "targeted" },
+  { label: "Traffic", value: "traffic" },
+  { label: "I-9 Audit", value: "i9" },
+  { label: "Checkpoint", value: "checkpoint" },
+  { label: "Action", value: "action" },
+  { label: "False Alarm", value: "falsealarm" },
+  { label: "Other", value: "other" },
+];
+
 export default class EventPage extends Component {
   static navigationOptions = () => ({ title: "Event Page" });
   state = {
     event: this.props.navigation.state.params.event
   };
 
+  getEventLabel(eventType) {
+    return types.find((type)=> {
+      return type.value == eventType;
+    }).label;
+  }
+
   render() {
     const { event } = this.state;
     const { location } = event;
+    const agencies = event.present && event.present.length > 0 ? (
+      <View>
+        <Text style={{paddingTop: 15, color: "gray"}}>Agencies:</Text>
+        <Text>{event.present.map((item)=> {return item.agency}).join(", ")}</Text>
+      </View>
+    ) : <></>
+
     return (
-      <View style={styles.container}>
-        <Text>{event.description}</Text>
-        <Text>{location.description}</Text>
-        <Text>{location.address_1}</Text>
-        <Text>{location.city}</Text>
-        <Text>{location.state}</Text>
-        <Text>{location.zipcode}</Text>
+      <View style={styles.view}>
+        <Container>
+          <Content style={styles.content}>
+            <H1 style={{paddingBottom: 15}}>{this.getEventLabel(event.type)}</H1>
+            <Text>{new Date(event.created.at).toLocaleString('en-US', {
```

Comment:
Is this where we would insert the device's selected locale?

Reply:
If this is what you mean-- 'en-US' will hopefully get subbed with a locale variable once we do translations.

Reply:
It is what I meant!